modules/components/analytics/config/org.wso2.carbon.pc.analytics.config/src/main/java/org/wso2/carbon/pc/analytics/config/clients/StreamAdminServiceClient.java

```diff
@@ -0,0 +1,98 @@
+import org.wso2.carbon.pc.analytics.config.stubs.EventStreamAdminServiceStub.AddEventStreamDefinitionAsString;
+import org.wso2.carbon.pc.analytics.config.stubs.EventStreamAdminServiceStub.AddEventStreamDefinitionAsStringResponse;
+
+import java.rmi.RemoteException;
+
+public class StreamAdminServiceClient {
+    private final String serviceName = "EventStreamAdminService";
+    private EventStreamAdminServiceStub serviceAdminStub;
+    private String endPoint;
+    ServiceClient serviceClient;
+    Options option;
+    private static final Log log = LogFactory.getLog(StreamAdminServiceClient.class);
+    JSONObject streamDefinitionJsonOb;
+
+    public StreamAdminServiceClient(String backEndUrl, String session, String streamName, String streamVersion,
+            String streamId, String streamNickName, String streamDescription, JSONArray processVariablesJObArr)
+            throws AxisFault {
+        this.endPoint = backEndUrl + "/services/" + serviceName;
+        serviceAdminStub = new EventStreamAdminServiceStub(endPoint);
+
+        // Authenticate stub from sessionCooke
+        serviceClient = serviceAdminStub._getServiceClient();
+        option = serviceClient.getOptions();
+        option.setManageSession(true);
+        option.setProperty(org.apache.axis2.transport.http.HTTPConstants.COOKIE_STRING, session);
+
+        streamDefinitionJsonOb = new JSONObject();
+        try {
+            streamDefinitionJsonOb.put("streamId", streamId);
+            streamDefinitionJsonOb.put("name", streamName);
+            streamDefinitionJsonOb.put("version", streamVersion);
+            streamDefinitionJsonOb.put("nickName", streamNickName);
+            streamDefinitionJsonOb.put("description", streamDescription);
+            JSONArray processVariablesJArr = new JSONArray();
+
+            //setting process variables as payloadData into the eventStream definition
+            streamDefinitionJsonOb.put("payloadData", processVariablesJObArr);
+            log.debug(streamDefinitionJsonOb.toString());
+        } catch (JSONException e) {
+            String errMsg = "Error in creating event stream Definition Json object";
+            log.error(errMsg, e);
+        }
+    }
+
+    public boolean createEventStream() {
+
+        AddEventStreamDefinitionAsString addEventStreamDefinitionAsString = new AddEventStreamDefinitionAsString();
+        AddEventStreamDefinitionAsStringResponse addEventStreamDefinitionAsStringResponse = null;
+        //addEventStreamDefinitionAsString
+        //		.setStreamStringDefinition("{ \"streamId\": \"streamId\",  			  \"name\": \"org.wso2.test1\",    			  \"version\": \"1.0.0\",    			  \"nickName\": \"TestStream\",    			  \"description\": \"Test Stream\", \"metaData\": [    {      \"name\": \"ip\",      \"type\": \"STRING\"    }  ],  \"correlationData\": [    {      \"name\": \"id\",      \"type\": \"LONG\"    }  ],  \"payloadData\": [    {      \"name\": \"testMessage\",    \"type\": \"STRING\"   }  ]}");
```

Comment:
Thanks @Niranjan-K. This is already fixed with da118e5e29f041d612ef4a6a53402148b1074b42 and 709fb21f7e4cf33e3cbb730270dc529ff1bbb8cc